config/locales/en/models.yml

```diff
@@ -184,6 +189,9 @@ en:
       user_identity:
         one: User identity
         other: User identities
+  task_contribution:
```

Comment:
I would have assumed that the plural form is advisable here (and in the German locale, of course). Why is it not?